models/equiformer_v3/test_equiformer_v3_kappa.py

```
@@ -0,0 +1,341 @@
+    df_ml = pd.read_json(pred_file).set_index("material_id")
+
+    json_path = DataFiles.phonondb_pbe_103_kappa_no_nac.path
+    df_dft = pd.read_json(json_path).set_index("material_id")
+
+    df_ml_metrics = phonons.calc_kappa_metrics_from_dfs(df_ml, df_dft)
+
+    kappa_srme_list = list(df_ml_metrics[Key.srme])
+    kappa_sre = df_ml_metrics[Key.sre].mean()
+    kappa_srme = df_ml_metrics[Key.srme].mean()
+
+    dir = os.path.dirname(path)
```

Ruff (0.15.9)

[error] Variable `dir` is shadowing a Python builtin

(A001)